bin/git-summary

```diff
@@ -192,6 +192,9 @@ uncommitted_changes_count() {
   git status --porcelain | wc -l
 }
 
+stashed_changes_count() {
+  git stash list | wc -l
```

Comment:
Now the output is:
```
commits     : 1802
 files       : 345
 uncommitted :        0
 stashed     :        0
```

There is an extra tab before the 0.
Could you remove the tab prefix of the output? Also to the one in uncommitted_changes_count.